### ISSUE TITLE
Hi, how to deploy in Netlify?

### ISSUE THREAD

**Issue by amirulasyraf88** [DivanteLtd/woocommerce2vuestorefront#5]
Hi, how to deploy in Netlify? Do you have documentation for deploying in Netlify?

**Comment by DragonEnergy** [comment]
Dont think  you can just deploy to netlify. You still need to run woocommerce to process transactions and manage products.

**Comment by amirulasyraf88** [comment]
yes I have the woocommerce running on other server .. already on production mode but seeking the fresh front end to start able to be flexible on mobile screen.
